Fix clang-4 warnings

With clang-4 I get:
```
[ 94%] Building CXX object xtp/src/libxtp/CMakeFiles/votca_xtp.dir/qmpackages/nwchem.cc.o
cd /home/votca/votca/build/xtp/src/libxtp && /usr/lib64/ccache/clang++  -DBOOST_UBLAS_NDEBUG -DGWBSE_DOUBLE=0 -Dvotca_xtp_EXPORTS -I/home/votca/votca/build/xtp/include -I/home/votca/votca/xtp/include -I/home/votca/votca/tools/include -I/home/votca/votca/build/tools/include -I/home/votca/votca/csg/include -I/home/votca/votca/moo/include -I/home/votca/votca/ctp/include -I/home/votca/votca/build/xtp/src/libxtp  -O2 -DNDEBUG -fPIC   -std=c++11 -o CMakeFiles/votca_xtp.dir/qmpackages/nwchem.cc.o -c /home/votca/votca/xtp/src/libxtp/qmpackages/nwchem.cc
In file included from /home/votca/votca/xtp/src/libxtp/qmpackages/nwchem.cc:22:
In file included from /home/votca/votca/xtp/include/votca/xtp/qminterface.h:36:
In file included from /home/votca/votca/xtp/include/votca/xtp/espfit.h:24:
/home/votca/votca/xtp/include/votca/xtp/elements.h:83:42: warning: comparison of unsigned expression >= 0 is always true [-Wtautological-compare]
        for ( unsigned i=temp.size2()-1;i>=0;i-- ){
                                        ~^ ~
1 warning generated.
```

--- a/include/votca/xtp/elements.h
+++ b/include/votca/xtp/elements.h
@@ -80,7 +80,7 @@ public:
         //cout<<temp<<endl;
         
         
-        for ( unsigned i=temp.size2()-1;i>=0;i-- ){
+        for ( int i=temp.size2()-1;i>=0;i-- ){
             //cout <<i<<endl;
             if(temp(0,i)<1) continue; 
             for (unsigned j=0;j<temp.size1();j++){

